app/components/Project/About/About.js

```diff
@@ -20,10 +22,12 @@ import TextEditor from '../../TextEditor/TextEditor';
 import LinkedDescription from '../LinkedDescription/LinkedDescription';
 import ProjectUpdateSummary from '../ProjectUpdateSummary/ProjectUpdateSummary';
 import { DescriptionContentType } from '../../../constants/constants';
+import { SplitMarkdownSections } from '../../../utils/SplitMarkdownSections';
 
 function About(props) {
   const [editing, setEditing] = useState(false);
   const [showCancelDialog, setShowCancelDialog] = useState(false);
+  const [collapsedSections, setCollapsedSections] = useState({});
```

Comment:
When the user has multiple projects, and they navigate between projects, it is preserving the collapsed sections across projects.  For example, ProjA has 3 section headings, ProjB has 5 section headings.  If I start at ProjA, and I collapse all the section headings, when I navigate to ProjB it has the first 3 section headings collapsed.  I feel like when the user switches projects we should reset the state.  I realize this means that if you go from ProjA -> ProjB -> ProjA all of the collapsed headings in ProjA would be reset.